### DISCUSSION TITLE
Ghost Tapping

### DISCUSSION THREAD

**Discussion by Beelijah** [FunkinCrew/Funkin#3780] [Ideas]
# The Great Idea.

You should add a feature in Options >> Preferences >> Ghost Tapping.

thats all

# its already developed

OH YEAH! the feature is already developed, too! in an html5 build of the game, search for "ghost_tapping = false;" in the file "Funkin.js", and change it to true.
then, you can launch the game, and play with ghost tapping!

# the end

the end

**Comment by NotHyper-474** [comment]
That has been rejected before https://github.com/FunkinCrew/Funkin/pull/3331#issuecomment-2357577868

**Comment by Beelijah** [comment]
aww.
well, if any other person reading this STILL wants to play with ghost tapping, you can use this!

repo:
https://github.com/Beelijah/fnfghosttapping.github.io

game:
https://beelijah.github.io/fnfghosttapping.github.io/

enjoy!
